### PR TITLE
Updated the generator to use the revised qt tool and the new schema org

### PR DESCRIPTION
**Pull request by michaelawyu**
DO NOT MERGE until [this PR](https://github.com/googleapis/google-cloudevents/pull/86) is merged.

### CODE REVIEW

File: gen/src/index.ts [hunk 1]
```diff
@@ -37,42 +33,52 @@ async function main() {
...
-    Object.keys(schema.properties).map((eventName: string) => {
-      const eventDescription = schema.properties[eventName].description;
-      pkgEvents.push(eventName);
-      allEvents.push({
-        package: pkg,
-        eventName: eventName,
-        eventDescription: eventDescription,
-      });
-    });
+    const eventDescription = schema.description;
```
Comment (grant) [inline gen/src/index.ts:48]: Can you document this file a bit more.

Comments for every main section/block would be nice.

File: gen/src/index.ts [hunk 2]
```diff
@@ -37,42 +33,52 @@ async function main() {
...
+      const pkgEvents: Event[] = [event];
+      allEventsByPkg.set(pkg, pkgEvents);
+    }
+  });
+
+  // Write __init__.py scripts
+  const initPySqrlTmpl = fs.readFileSync(
+    `${templateDirectoryPath}/${INIT_PY_TEMPLATE}`
+  );
+  for (const k of allEventsByPkg.keys()) {
```
Comment (grant) [inline gen/src/index.ts:68]: Spell out keys. Or even better, spell out what the the keys represent.

`packageKey`?

File: gen/src/index.ts [hunk 3]
```diff
@@ -37,42 +33,52 @@ async function main() {
...
+    const pkgEvents = allEventsByPkg.get(k);
+    const pkgPath = k.replace(/\./g, '/');
```
Comment (grant) [inline gen/src/index.ts:70]: An example would be nice.

**Review by grant — APPROVED** [review]
- Can we spell out all variables? This will make the code easier to read.

Looks good otherwise.

File: gen/src/index.ts (hunk 2 above)
Comment (michaelawyu) [reply gen/src/index.ts]: Done.

**Pull request merged**